orchestration/hca_manage/validation.py

```diff
@@ -1,97 +1,38 @@
-import json
 import logging
-from functools import cache
-from typing import Optional, Any
-
-import requests
-from google.cloud import storage
-from jsonschema import validate
+from typing import Any, Optional
 
 from hca_manage.common import DefaultHelpParser
-from hca_orchestration.contrib.gcs import parse_gs_path
 from hca.staging_area_validator import StagingAreaValidator
 
 
-@cache
-def fetch_schema(path: str) -> Any:
-    raw_schema = requests.get(path).json()
-    return raw_schema
-
-
-def validate_json(blob: storage.Blob) -> Optional[Exception]:
-    """
-    Validate that the JSON file blob follows the schema in the describedBy
-    :param blob: JSON file blob to check
-    :param schema_fetcher: Schema Fetcher that contains a set of unique schemas
-    :return: An exception if JSON is invalid, return None if valid
-    """
-    try:
-        data = json.loads(blob.download_as_string())
-        schema_path = data["describedBy"]
-        schema = fetch_schema(schema_path)
-        validate(instance=data, schema=schema)
-        return None
-    except Exception as e:
-        return e
-
-
-def validate_directory(path: str, bucket: storage.Client.bucket) -> None:
-    """
-    Validate that the directory in the Google storage bucket is valid
-    :param path: Google stage path name
-    :param bucket: Google storage bucket name
-    """
-    valid_files_in_dir = []
-    invalid_files_in_dir = {}
-    for blob in bucket.list_blobs(prefix=path):
-        if blob.name.endswith('json'):
-            json_error = validate_json(blob)
-            if json_error is None:
-                valid_files_in_dir.append(blob.name)
-            else:
-                invalid_files_in_dir[blob.name] = json_error
+class HcaValidator:
+    def validate_staging_area(self, path: str, ignore_inputs: bool) -> Any:
```

Comment:
Updated the return type to "Any" here because of a failing mypy check:
`error: Returning Any from function declared to return "int"`